src/components/TodoList.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { TodoContext } from '../contexts/TodoContext';
+import { deleteTodo } from '../api/todos';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+export const TodoList: React.FC = () => {
+  const {
+    filteredTodos,
+    setTodos,
+    setErrorMessage,
+  } = useContext(TodoContext);
+
+  const handleCatch = () => {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  };
```

Comment:
when you move this logic according to my previous comment you can avoid this code duplication with setTimeout